fl4health/clients/basic_client.py

```diff
@@ -893,13 +894,14 @@ def set_optimizer(self, config: Config) -> None:
         self.optimizers = {"global": optimizer}
 
     def clone_and_freeze_model(self, model: nn.Module) -> nn.Module:
-        """Clone and freeze model for use in various loss calculation.
+        """
+        Clones and freezes a models weights. This is to preserve the model in its current state while decoupling in
```

Comment:
Maybe its just me but the way this is worded is a tad confusing. Perhaps: Creates a clone of the model with frozen weights to be used in loss calculations so the original model is preserved in its current state.

Reply:
Sounds good!